keyboards/lily58/keymaps/default/keymap.c

```diff
@@ -0,0 +1,154 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define QWERTY 0
+#define LOWER 1
+#define RAISE 2
+//#define RGB 3
+
+// Fillers to make layering more clear
```

Comment:
This code block (Lines 10-12) isn't actually necessary.

These keycodes are defined automatically by QMK, so there's no need to put the definitions here.

https://github.com/qmk/qmk_firmware/blob/ce122c4981c71d33a85db94c787d5ec7a823acc6/quantum/quantum_keycodes.h#L29-L31